flume/doc/operation/hdfs_kerberos_authentication.md

```diff
@@ -0,0 +1,67 @@
+# Enabling Kerberos SPNEGO authentication in HDFS
```

Comment:
Maybe the architecture.md document should be update to include new diagrams related with Kerberos? In theory, that document is used to identify flows between "third party" system (in this case, the Kerberos server), so probably it makes sense.